kong/tools/wrpc/future.lua

```diff
@@ -0,0 +1,87 @@
+local semaphore = require "ngx.semaphore"
+local semaphore_new = semaphore.new
+
+local ngx_log = ngx.log
+local ERR = ngx.ERR
+local ngx_now = ngx.now
+local new_timer = ngx.timer.at
+local setmetatable = setmetatable
+
+local _M = {}
+local _MT = { __index = _M, }
+
+-- this future is no longer tracked
+-- don't call this if you don't know what this means
+function _M:finish()
+  self.response_t[self.seq] = nil
+end
+
+local function dropWait(permature, future)
```

Comment:
We use snake_case, not camelCase, this should be `drop_wait`.

Reply:
Fixed.